apps/server/src/modules/common-cartridge/common-cartridge-client/card-client/mapper/card-response.mapper.spec.ts

```diff
@@ -0,0 +1,216 @@
+import { faker } from '@faker-js/faker';
+import { CardResponseMapper } from './card-response.mapper';
+import {
+	CardListResponse,
+	DeletedElementResponse,
+	SubmissionContainerElementResponse,
+	DrawingElementResponse,
+	ExternalToolElementResponse,
+	FileElementResponse,
+	LinkElementResponse,
+	RichTextElementResponse,
+	CollaborativeTextEditorElementResponse,
+	CardResponseElementsInner,
+} from '../cards-api-client';
+import { ContentElementType } from '../enums/content-element-type.enum';
+import { CardContentElementInner } from '../types/card-content-elements-inner.type';
+import { CardResponseDto } from '../dto/card-response.dto';
+
+describe('CardResponseMapper', () => {
```

Comment:
use please faker for dummy data in tests

Reply:
Done. Still using constants where necessary to ascertain response correctness.